clippy_lints/src/utils/conf.rs

```diff
@@ -551,6 +551,16 @@ define_Conf! {
     ///
     /// Whether to allow `r#""#` when `r""` can be used
     (allow_one_hash_in_raw_strings: bool = false),
+    /// Lint: ABSOLUTE_SYMBOL_PATHS.
+    ///
+    /// The maximum number of segments a path can have before being linted, anything above this will
+    /// be linted.
+    (absolute_paths_max_segments: u64 = 2),
+    /// Lint: ABSOLUTE_SYMBOL_PATHS.
```

Comment:
```suggestion
    /// Lint: ABSOLUTE_PATHS.
    ///
    /// The maximum number of segments a path can have before being linted, anything above this will
    /// be linted.
    (absolute_paths_max_segments: u64 = 2),
    /// Lint: ABSOLUTE_PATHS.
```